charts/alfresco-repository/templates/_helpers-database.tpl

```diff
@@ -4,7 +4,7 @@ We're just manipulating the string URl to make it parseable by urlParse.
 It MUST NOT be used directly.
 */}}
 {{- define "alfresco-repository.jdbc.parser" -}}
-{{- $jdbc_url := required "Alfresco repository needs a database to start. Please provide a valid URL in db.url value" . }}
+{{- $jdbc_url := required "Alfresco repository needs a database to start. Please provide a valid URL in configuration.db.url value" . }}
```

Comment:
this error message is not too specific for what this template is doing? I don't expect any values referenced here

if a more specific message is necessary then it needs to be moved before calling this template?

Reply:
wait... actually this template should not even exist in this chart anymore. It's part of `alfresco-common` chart.
Will remove these